### DISCUSSION TITLE
Sensor over VPN

### DISCUSSION THREAD

**Discussion by Shorider** [Security-Onion-Solutions/securityonion#5970] [General] [answered]
Good morning everyone!

I just had a quick question regarding our network we are setting up.  We are trying to keep as much physical hardware as we can on our site while setting up a backup in a different location.  We are trying to determine if we can have a sensor at our backup location sending data through VPN to our logger/master on our main site.

Thoughts?

**Comment by dougburks** [comment, marked as answer]
Yes, you should be able to use VPN:
https://github.com/Security-Onion-Solutions/securityonion/issues/1139
